index.html

```diff
@@ -571,70 +571,98 @@ <h3>External Binary Content</h3>
+          content. Fedora servers that do not support the creation of <a>LDP-NR</a>s with content external
+          to the request entity MUST reject such requests with a 4xx range status code. Fedora servers MUST
+          reject with a 4xx range status code requests that include multiple
+          <code>rel="http://fedora.info/definitions/fcrepo#ExternalContent"</code> links.
         </p>
-        <p id='message-external-body-response-headers'>
-          Fedora servers receiving requests that would create or update an <a>LDP-NR</a> with
-          <code>Content-Type: message/external-body</code> MUST NOT accept the request if it cannot guarantee all of
-          the response headers required by the <a>LDP-NR</a> interaction model in this specification.
+        <p id='external-content-handling'>
+          Fedora servers MUST use the <code>handling</code> attribute in the external content link to determine
+          how to process the request. This specification defines the following attibutes and expected behaviors:
+        <p>
+        <ul>
+          <li><code>copy</code> - requests that the server dereference the external content URI and treat that as if
+            it were the entity body of the request.</li>
+          <li><code>redirect</code> - requests that the server record the location of the external content and
+            handle requests for that content using HTTP redirect responses with the <code>Content-Location</code>
+            header specifying the external content location. See also <a href="#redirect-and-proxy"></a>.</li>
+          <li><code>proxy</code> - requests that the server  record the location of the external content and
+            handle requests for that content by proxying. See also <a href="#redirect-and-proxy"></a>.</li>
+        </ul>
+        <p>
+          Fedora servers MUST reject with a 4xx range status code requests for which the <code>handling</code>
+          attribute is not present or cannot be respected.
         </p>
-        <p id='external-content-content-location'>
-          <code>GET</code> and <code>HEAD</code> responses for any external <a>LDP-NR</a> SHOULD include a
-          <code>Content-Location</code> header with a URI representation of the location of the external content if
-          the Fedora server is proxying the content.
+        <p id='external-content-media-type'>
+          Fedora servers MUST use the value of the <code>type</code> attribute in the external content link as
+          the media type of the external content, if provided. Otherwise:
         </p>
-        <p id='external-content-want-digest'>
-          <code>GET</code> and <code>HEAD</code> requests to any external <a>LDP-NR</a> MUST correctly
-          respond to the <code>Want-Digest</code> header defined in [[!RFC3230]].
+        <ul>
+          <li>Servers MAY use the media type specified in a <code>Content-Type</code> header of the request.
```

Comment:
So, is there any semantic difference between a request that contains a `Content-Type` but not a `type=` vs a request that omits `Content-Type` but has a `type=`?

If there is no difference, then why is it explicitly spelled out that a server is allowed to ignore a provided `Content-Type` header for external content?   

I can see that having two ways of doing the same thing would be confusing and problematic, especially if both `Content-Type` and `type=` had a value that was different.  Defining a preferred order of precedence is fine, as is picking one over the other.  The spec, though, suggests that in the absence of `type=`, the server can explicitly ignore `Content-Type`, which does not seem intuitive, as it's not explicitly allowed to ignore it in any other context.

Reply:
I think we should remove the bullet about using `Content-Type` and instead say that _"Implementations SHOULD ignore any `Content-Type` header on external content requests."_. There seems only to be confusion, but no benefit, from having two ways of specifying

Reply:
Yes, that would be more clear. The question as to whether there is a semantic difference between `Content-Type` and `type=` is important.  If there _is_, then it might be worthwhile strengthening that to a _MUST ignore any `Content-Type`_.